examples/pipelines/finetune_stable_diffusion/components/embedding_component/src/main.py

```diff
@@ -0,0 +1,86 @@
+"""
+This component adds a data source to the manifest by embedding the images.
+"""
+import logging
+from typing import Optional, Union, Dict
+
+from tqdm.auto import tqdm
+
+from express.components.hf_datasets_components import HFDatasetsTransformComponent, HFDatasetsDataset, HFDatasetsDatasetDraft
+from express.logger import configure_logging
+
+import torch
+
```

Comment:
can you group dependencies together? you should get a warning once you enable linting here